pytypes/typechecker.pyi

```diff
@@ -0,0 +1,48 @@
+# Generated using mypy stubgen, then editing types for typechecked and
+# no_type_check.
+
+import __builtin__ as builtins
+import types
+import typing
+from . import type_util as type_util, util as util
+from .exceptions import InputTypeError as InputTypeError, OverrideError as OverrideError, ReturnTypeError as ReturnTypeError
+from .type_util import TypeAgent as TypeAgent, deep_type as deep_type, get_orig_class as get_orig_class, has_type_hints as has_type_hints, is_Tuple as is_Tuple, is_builtin_type as is_builtin_type, type_str as type_str
+from .util import getargspecs as getargspecs
+from typing import Any, Optional, Callable, TypeVar, Union, overload
+from warnings import warn as warn
+
```

Comment:
In the import section, please clean up all ``blah as blah`` to be just ``blah``. I guess this is a auto-generated pattern.

Reply:
Yeah, removed. I initially left it that way because that's what stubgen had generated and I figured it was potentially safer (if there was a reason they'd done that) and less maintenance overhead to leave it as-is.